lxd/device/disk.go

```diff
@@ -1637,13 +1637,10 @@ func (d *disk) mountPoolVolume() (func(), string, *storagePools.MountInfo, error
 // createDevice creates a disk device mount on host.
 // The srcPath argument is the source of the disk device on the host.
 // Returns the created device path, and whether the path is a file or not.
-func (d *disk) createDevice(srcPath string) (func(), string, bool, error) {
+func (d *disk) createDevice(srcPath string, devPath string) (func(), bool, error) {
```

Comment:
as an aside, the createDevice function returns a revert hook `func()` but to make this clearer please can you change the type of the return value to `revert.Hook`.